### PR TITLE
Add bash debugger.

### PR DESCRIPTION
**Pull request by AndrienkoAleksandr**
### What does this PR do?
Add bash debugger. Useful for dog fooding.

Signed-off-by: Oleksandr Andriienko <oandriie@redhat.com>

### CODE REVIEW

**Comment by benoitf** [comment]
@AndrienkoAleksandr you'll need to fix the build before the merge

there is no latest.txt in your commit

File: v3/plugins/rogalmic/vscode-bash-debug/0.3.9/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+name: bash-debug
+version: 0.3.9
+type: VS Code extension
+displayName: Bash Debug
+title: A bash debugger GUI frontend based on awesome bashdb scripts (bashdb now included in package).
+description: This is a SIMPLE bashdb debugger frontend. Useful for learning bash shell usage and writing simple scripts.
+icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
+repository: https://github.com/rogalmic/vscode-bash-debug
+category: Other
+firstPublicationDate: '2020-07-4'
```
Comment (benoitf) [inline v3/plugins/rogalmic/vscode-bash-debug/0.3.9/meta.yaml:12]: maybe use the 2 digits (2020-07-04)
```suggestion
firstPublicationDate: '2020-07-04'
```

**Comment by benoitf** [comment]
@AndrienkoAleksandr and please add this item to https://github.com/eclipse/che-plugin-registry/blob/master/vscode-extensions.json

**Comment by ericwill** [comment]
Can you add an entry to the [vscode-extension.json](https://github.com/eclipse/che-plugin-registry/blob/master/vscode-extensions.json) plugin list. For example in this case the entry would be:

```js
{
  "repository": "https://github.com/rogalmic/vscode-bash-debug",
  "revision": "v0.3.9"
}
```

You can look at the other entries there for an idea.

**Comment by benoitf** [comment]
@AndrienkoAleksandr 
```
---> Running in 23446521decb
!!! Location mismatch in plugin 'rogalmic/bash-debug/latest':
!!!   Expected location: 'v3/plugins/rogalmic/bash-debug/latest/meta.yaml'
!!!   Actual location: 'v3/plugins/rogalmic/vscode-bash-debug/latest/meta.yaml' 
```

**Comment by benoitf** [comment]
let's squash and merge 🎉

**Pull request merged**